index.html

```diff
@@ -25,14 +25,17 @@ <h1>ROCK SCISSORS PAPER</h1>
                     <p class="points">Points: <b id="user-points">0</b></p>
 
                     <div class="images">
-                        <img src="img/paper.png" alt="paper" height="170px" width="170px">
-                        <br>
-                        <img src="img/rock.png" alt="rock" height="170px" width="170px">
+                        <button class="user-choice" id="paper"><img src="img/paper.png" alt="paper" height="170px" width="170px"></button>                       
                         <br> <br>
-                        <img src="img/scissors.png" alt="scissors" height="170px" width="170px">
+                        <button class="user-choice" id="rock"><img src="img/rock.png" alt="rock" height="170px" width="170px"></button>
+                        <br> <br>
+                        <button class="user-choice" id="scissors"><img src="img/scissors.png" alt="scissors" height="170px" width="170px"></button>                     
                     </div>
 
-                    <div id="choosen" class="choosen"></div>
+                    <div class="choosen" id="choosen-paper"><img src="img/paper.png" alt="paper" height="170px" width="170px"></div>
+                    <div class="choosen" id="choosen-rock"><img src="img/rock.png" alt="rock" height="170px" width="170px"></div>
+                    <div class="choosen" id="choosen-scissors"><img src="img/scissors.png" alt="scissors" height="170px" width="170px"></div>
+                   
```

Comment:
redundant empty line. The same with the `width` and `height` as before (we could use the same class). I think `alt` should also indicate that this is a chosen image.